src/classes/GE_BatchGiftEntryController_TEST.cls

```diff
@@ -0,0 +1,70 @@
+@isTest
+private with sharing class GE_BatchGiftEntryController_TEST {
+
+    /*********************************************************************************************************
+    * @description Sets up a new NPSP_Data_Import_Batch__c Record
+    * @return Id of the new NPSP_Data_Import_Batch__c
+    */
+    private static Id createDataImportBatch( String batchName ) {
```

Comment:
I saw similar methods being created in other Tests, but I didn't want to introduce circular dependencies so I created the ones I needed for my tests.
 
I think we should have a single unit test helper in charge of creating ALL the GiftEntry assets (di, batches, etc) or at least to act as a hub and only call that class from our unit tests. But that's require a lot of change at this moment

Reply:
Yes, good points - I generally agree, although we should also think about when and if we should build our tests in a way that is less dependent on data. It's less intuitive but having been around this product for a year I can really see the advantages to test stability and execution time. I think this is probably good for this delivery!